lib/reality/extras/economic.rb

```diff
@@ -0,0 +1,103 @@
+require 'quandl'
+Quandl::ApiConfig.api_key = 'JU6KEPEkqqqfVkcpa9MU'
+Quandl::ApiConfig.api_version = '2015-04-09'
```

Comment:
I feel like we should stop this practice immediately :(
There should be some configuration block, and some `config/demo.yml`, and `Reality.configure(path)`